pipelines/rj_smas__disparo_pic/flow.py

```
@@ -1,33 +1,41 @@
 # -*- coding: utf-8 -*-
-# noqa: flake8=E501
+# flake8: noqa:E501
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)